### ISSUE TITLE
Add archive link to Zenodo releases

### ISSUE THREAD

**Issue by DeniseSl22** [bridgedb/bridgedb.github.io#10]
@tabbassidaloii  @egonw  @DeniseSl22

**Comment by egonw** [comment]
Can you elaborate? We already have such links for the data. Do you mean for the Java library, etc?

**Comment by tabbassidaloii** [comment]
> Can you elaborate? We already have such links for the data. Do you mean for the Java library, etc?

There are only links to the most recent versions of derby files on Zenodo, not the old releases. 
We suggest archiving the Zenodo links for all different releases (e.g. links to v103, v104, etc.).

**Comment by DeniseSl22** [comment]
@egonw : If I want to use an older version of mapping files, where can I find them now on the BridgeDb website (besides going to GitHub and going through the version history?)

**Comment by egonw** [comment]
Yes, I understand the use case now. I don't have a solution right now.
